### ISSUE TITLE
Dependencies missing in install documentation

### ISSUE THREAD

**Issue by dublutz** [capacity4dev/capacity4more#529]
I'm installing capacity4home profile on my local and so far I have 2 errors because of dependencies that I don't have installed. 

One is Grunt, the other one is Compass. 

Can you add the dependencies in the install procedure? 

Thanks!

**Comment by amitaibu** [comment]
Yes, we should improve the readme (bonus points if you will create a PR ;)

You need to:
```
gem install compass
npm install -g grunt-cli
```

‫ב-27 בפבר 2015, בשעה 15:48, ‏dublutz <notifications@github.com> כתב/ה:‬
